Directory.Build.props

```diff
@@ -80,7 +80,7 @@
     <PackageProjectUrl>https://github.com/dotnet/maui</PackageProjectUrl>
     <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
-    <PackageVersion>0.0.1-alpha1</PackageVersion>
+    <PackageVersion>6.0.100-dev</PackageVersion>
```

Comment:
If you build dotnet/installer locally, it has this version number.

I thought we could put the same here, so this check would pass:

https://github.com/dotnet/maui/blob/887e65e67505bd0bfb1e050043da5d29f760d4bc/.nuspec/Microsoft.Maui.Controls.targets#L45-L47